robocopkaka/house-renting-app
# rewrite models with typescript

#### What does this PR do?
Rewrites existing models in Typescript
#### Description of Task to be completed?
  - rewrite models using typescript
  - move index.ts to root of src/
  - modify src/cpntrollers/userController.ts
  - modify tests/users.ts
  - initialize sequelize in src/app.ts
  - import models directly
  - rename model files to start with uppercase
  - install `sequelize-typescript`
#### How should this be manually tested?
- run `yarn clean`
- run `yarn build`
- run `yarn test`
- Tests should pass
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories? (if applicable)
https://www.pivotaltracker.com/story/show/172215185
